cli/cache/disk_cache.rs

```diff
@@ -1,12 +1,13 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
+use crate::sys::CliSys;
```

Comment:
Shouldn't this one be under all the other imports? 😏

Reply:
Let's merge that other fmt pr 😅

I'll rebase it in a few minutes.